modules/consentManagement.js

```diff
@@ -322,6 +325,13 @@ function processCmpData(consentObject, hookConfig) {
   // determine which set of checks to run based on cmpVersion
   let checkFn = (cmpVersion === 1) ? checkV1Data : (cmpVersion === 2) ? checkV2Data : null;
 
+  // Raise deprecation warning if 'allowAuctionWithoutConsent' is used with TCF 2.
+  if (allowAuction.definedInConfig && cmpVersion === 2) {
+    utils.logWarn(`consentManagement config is using deprecated property 'allowAuctionWithoutConsent' with TCF 2 setup. This property only works with TCF 1.1`);
```

Comment:
I'm on a "shorten log messages to save space" mission lately. Suggest: "allowAuctionWithoutConsent ignored for TCF2"